client/src/app/site/pages/meetings/services/media-manage.service.ts

```diff
@@ -18,46 +21,31 @@ import { ActiveMeetingService } from './active-meeting.service';
 })
 export class MediaManageService {
     public get allLogoPlaces(): LogoPlace[] {
-        return [
-            `projector_main`,
-            `projector_header`,
-            `web_header`,
-            `pdf_header_l`,
-            `pdf_header_r`,
-            `pdf_footer_l`,
-            `pdf_footer_r`,
-            `pdf_ballot_paper`
-        ];
+        return LogoPlaces;
     }
 
     public get allFontPlaces(): FontPlace[] {
-        return [
-            `regular`,
-            `italic`,
-            `bold`,
-            `bold_italic`,
-            `monospace`,
-            `chyron_speaker_name`,
-            `projector_h1`,
-            `projector_h2`
-        ];
+        return FontPlaces;
     }
 
     private readonly logoUrlSubjects: { [place in LogoPlace]?: BehaviorSubject<string | null> } = {};
     private readonly fontUrlSubjects: { [place in FontPlace]?: BehaviorSubject<string> } = {};
 
     public constructor(
         private activeMeetingService: ActiveMeetingService,
-        private mediaAdapter: MeetingMediaAdapterService
+        private mediaAdapter: MeetingMediaAdapterService,
+        private mediaRepo: MediafileRepositoryService
     ) {
-        this.activeMeetingService.meetingObservable.subscribe(_ => {
-            for (const place of Object.keys(this.logoUrlSubjects)) {
-                (<any>this.logoUrlSubjects)[place].next(this.getLogoUrl(place as LogoPlace));
+        merge(this.activeMeetingService.meetingObservable, this.mediaRepo.getViewModelListUnsafeObservable()).subscribe(
+            _ => {
+                for (const place of Object.keys(this.logoUrlSubjects)) {
+                    (<any>this.logoUrlSubjects)[place].next(this.getLogoUrl(place as LogoPlace));
```

Comment:
Can you improve this typing? Casting to `any` is always a bad idea, maybe you could e.g. cast `place` to `LogoPlace` first, or what is the problem why the typing isn't recognized correctly without casting?